### ISSUE TITLE
[CTA section]: QA: Heading text is not given in heading tag.

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9710]
### Description

Page Element:
The following content present after the first heading in the component: Optional text heading-03 color text-01, Lorem ipsum dolor sit amet, consecteture adipiscing elit sed dose. 

Issue Description:
The mentioned text is not given in heading tag.

Recommend Fix:
If it is supposed to be a heading, Provide appropriate heading level.

User Impact:
When structural markups like headings and lists are not used, Screen readers cannot convey the semantic information of those elements. This impacts users who are blind

Tool:
JAWS_Desktop
Voiceover_iPhone

### Component(s) impacted

CTA section

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
